skynet/modules/stt/vox/app.py

```diff
@@ -0,0 +1,72 @@
+import base64
+
+from fastapi import FastAPI, WebSocket, WebSocketDisconnect
+
+from skynet.logs import get_logger
+from skynet.modules.stt.streaming_whisper.utils import utils
+from skynet.modules.stt.vox.connection_manager import ConnectionManager
+from skynet.modules.stt.vox.decoder import PcmaDecoder
+
+log = get_logger(__name__)
+
+ws_connection_manager = ConnectionManager()
+app = FastAPI()
+decoder = PcmaDecoder()
```

Comment:
In general, it makes more sense to use a decoder per connection, since they might be using different sample rates / codecs and audio samples would interleave.

Reply:
Even if that's not the case yet, let's make it a local variable in the websocket handler please.